src/style.css

```diff
@@ -2,0 +1,32 @@
+* {
+  margin: 0;
+}
+
+html {
+  font-size: 10px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 1.6rem;
+  height: 1.6rem;
+  margin-right: 4px;
```

Comment:
Be consistent with your margins. Add only top or bottom margins, but not both, to avoid potential margin collapse issues.